src/frontend/components/Filters/Filters.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import { filterCollections } from "../../data";
+import { FilterCollection } from "../FilterCollection/FilterCollection";
+import "./Filters.css";
+
+const Filters = () => {
+  const [showResponsiveFilters, setShowResponsiveFilters] = useState(false);
+
+  useEffect(() => {
+    const checkResponsiveFilters = () => {
+      if (window.innerWidth >= 1200) {
+        setShowResponsiveFilters(false);
+      }
+    };
+    window.addEventListener("resize", checkResponsiveFilters);
+  }, []);
+
+  return (
+    <>
+      <div
+        className="products-filter-responsive"
+        onClick={() => setShowResponsiveFilters(!showResponsiveFilters)}
```

Comment:
do with use of CB as bool => !bool inside setState

Reply:
did it!